auto_update_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"time"
 )
 
-func ExampleTGFConfig_RunWithUpdateCheck_forcing_update_cli() {
+func ExampleRunWithUpdateCheck_forcing_no_update_cli() {
```

Comment:
You should use a mock here. Here, you are fetching the version from github and you are also using local files to verify update time. You could cover 100% of that code of that function's code using the mock